# German po for libgda
# Copyright (C) 2000 Free Software Foundation, Inc.
# Kai Lahmann <kl@linuxfaqs.de>, 2000.
msgid ""
msgstr ""
"Project-Id-Version: libgda   \n"
"POT-Creation-Date: 2000-10-01 14:04+0200\n"
"PO-Revision-Date: 2000-10-01 15:31+0200\n"
"Last-Translator: Kai Lahmann <kl@linuxfaqs.de>\n"
"Language-Team: German <gnome-de@gnome.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=iso-8859-1\n"
"Content-Transfer-Encoding: 8bit\n"

#: lib/gda-client/gda-connection-pool.c:169
#, c-format
msgid "could not open connection to %s"
msgstr "konnte Vebindung zu %s nicht öffnen"

#: lib/gda-client/gda-connection-pool.c:180
#, c-format
msgid "Data source %s not found"
msgstr "Datenquelle %s nicht gefunden"

#: lib/gda-client/gda-connection.c:166 lib/gda-client/gda-error.c:116
#, c-format
msgid "%s: The server didn't respond."
msgstr "%s: Der Server antwortet nicht."

#: lib/gda-client/gda-connection.c:170 lib/gda-client/gda-error.c:120
#, c-format
msgid "%s: An Error occured in the CORBA system."
msgstr "%s: Ein Fehler ist in dem CORBA-System aufgetreten."

#: lib/gda-client/gda-connection.c:453
#, c-format
msgid "Database '%s' not found in system configuration"
msgstr "Datenbank '%s' nicht in der System-Konfuguration gefunden"

#: lib/gda-client/gda-connection.c:469
msgid "Could not open CORBA factory"
msgstr "Konnte CORBA-Fabrik nicht öffnen"

#: lib/gda-client/gda-field.c:187
msgid "<NULL>"
msgstr "<NULL>"

#: lib/gda-client/gda-field.c:193 lib/gda-client/gda-field.c:195
msgid "<Unknown GDA Type(NULL)>"
msgstr "<Unbekannter GDA-Typ(NULL)>"

#: lib/gda-client/gda-field.c:208
msgid "TRUE"
msgstr "WAHR"

#: lib/gda-client/gda-field.c:209
msgid "FALSE"
msgstr "FALSCH"

#: lib/gda-common/gda-corba.c:72
msgid "True"
msgstr "Wahr"

#: lib/gda-common/gda-corba.c:72
msgid "False"
msgstr "Falsch"

#: lib/gda-common/gda-corba.c:119
#, c-format
msgid "CORBA System exception: %s"
msgstr "CORBA-System-Fehler: %s"

#: lib/gda-common/gda-log.c:29
msgid "Saving log...x"
msgstr "Speichere Log...x"

#. compose message
#: lib/gda-common/gda-log.c:50
msgid "ERROR: "
msgstr "Fehler:"

#: lib/gda-common/gda-log.c:50
msgid "MESSAGE: "
msgstr "Meldung:"

#: lib/gda-common/gda-thread.c:150
msgid "thread is already running"
msgstr "Thread läuft schon"

#: lib/gda-common/gda-xml-database.c:224
#, c-format
msgid "table %s already exists"
msgstr "Tabelle %s existiert schon"

#: lib/gda-common/gda-xml-database.c:388
#, c-format
msgid "%p is not a valid table"
msgstr "%p ist keine gültige Tabelle"

#: lib/gda-server/gda-server-error.c:169
msgid "Could not get pointer to server implementation"
msgstr "Konnte keinen Pointer auf die Server-Implementiertung setzen"

#: lib/gda-server/gda-server-impl-connection.c:106
#: lib/gda-server/gda-server-impl-connection.c:115
#: lib/gda-server/gda-server-impl-connection.c:122
#: lib/gda-server/gda-server-impl-connection.c:131
#: lib/gda-server/gda-server-impl-connection.c:138
#: lib/gda-server/gda-server-impl-connection.c:147
#: lib/gda-server/gda-server-impl-connection.c:154
#: lib/gda-server/gda-server-impl-connection.c:163
#: lib/gda-server/gda-server-impl-recordset.c:113
#: lib/gda-server/gda-server-impl-recordset.c:122
#: lib/gda-server/gda-server-impl-recordset.c:129
#: lib/gda-server/gda-server-impl-recordset.c:144
#: lib/gda-server/gda-server-impl-recordset.c:153
#: lib/gda-server/gda-server-impl-recordset.c:160
#: lib/gda-server/gda-server-impl-recordset.c:169
#: lib/gda-server/gda-server-impl-recordset.c:176
#: lib/gda-server/gda-server-impl-recordset.c:185
#: lib/gda-server/gda-server-impl-recordset.c:192
#: lib/gda-server/gda-server-impl-recordset.c:200
#: lib/gda-server/gda-server-impl-recordset.c:209
#: lib/gda-server/gda-server-impl-recordset.c:216
#: lib/gda-server/gda-server-impl-recordset.c:224
#: lib/gda-server/gda-server-impl-recordset.c:232
#: lib/gda-server/gda-server-impl-recordset.c:274
#: lib/gda-server/gda-server-impl-recordset.c:284
#: lib/gda-server/gda-server-impl-recordset.c:292
#: lib/gda-server/gda-server-impl-recordset.c:300
#: lib/gda-server/gda-server-impl-recordset.c:308
#: lib/gda-server/gda-server-impl-recordset.c:316
#: lib/gda-server/gda-server-impl-recordset.c:325
#: lib/gda-server/gda-server-impl-recordset.c:333
#: lib/gda-server/gda-server-impl-recordset.c:341
#, c-format
msgid "%s not implemented"
msgstr "%s nicht implementiert"

#: lib/gda-server/gda-server-impl-connection.c:255
#, c-format
msgid "impl_GDA_Connection_open: opening connection with DSN: %s"
msgstr "impl_GDA_Connection_open: Öffne Verbindung mit DSN: %s"

#: lib/gda-server/gda-server-impl-recordset.c:240
#: lib/gda-server/gda-server-impl-recordset.c:248
#, c-format
msgid "%s: not implemented"
msgstr "%s: nicht implemeniert"

#: lib/gda-server/gda-server-impl-recordset.c:434
#, c-format
msgid "%s: an error ocurred while fetching data"
msgstr "%s: Ein Fehler trat auf, während die Daten empfangen werden"

#: lib/gda-server/gda-server-impl-recordset.c:473
#, c-format
msgid "%s: length = %d"
msgstr "%s: Länge = %d"

#: lib/gda-server/gda-server-impl-recordset.c:486
#, c-format
msgid "%s: sending field description for '%s'"
msgstr "%s: Sende Feld-Beschreibung für '%s'"

#: lib/gda-server/gda-server-impl.c:114
#, c-format
msgid "Starting provider %s with no implementation functions"
msgstr "Starte Anbieter %s ohne Implementierungs-Funktionen"

#: lib/gda-server/gda-server-impl.c:141
msgid ""
"OAF doesn't know about our IID; indicates broken installation. Exiting..."
msgstr ""
"OAF weiß nicht über ihre IID; Deutet auf defekte Installation hin. Beende..."

#: lib/gda-server/gda-server-impl.c:144
msgid ""
"Another instance of this provider is already registered with OAF. Exiting..."
msgstr ""
"Eine andere Instalz dieses Anbieters ist bereits mit OAF registriert. "
"Beende..."

#: lib/gda-server/gda-server-impl.c:147
msgid "Unknown error registering this provider with OAF. Exiting"
msgstr "Unbekannter Fehler beim registrieren dieses Anbieters mit OAF. Beende"

#: lib/gda-server/gda-server-impl.c:151
#, c-format
msgid "Registered with ID = %s"
msgstr "Registriert mit ID = %s"

#: lib/gda-server/gda-server-impl.c:241
#, c-format
msgid "CORBA system exception %s"
msgstr "CORBA-System-Fehler %s"

#: lib/gda-server/gda-server-impl.c:244
#, c-format
msgid "CORBA user exception: %s"
msgstr "CORBA Anwenderfehler: %s"

#: lib/gda-server/gda-server-init.c:39 lib/gda-server/gda-server-init.c:45
#, c-format
msgid "Received signal %d, shutting down"
msgstr "Erhiert Signal %d, schließe"

#: lib/gda-server/gda-server-init.c:49
#, c-format
msgid "Received signal %d. Ignoring"
msgstr "Erhiert Signal %d. Ignoriere"

#: lib/gda-server/gda-server-init.c:97
msgid "Attempt to initialize an already initialized provider"
msgstr "Versucht einen bereits initialisieren Anbieter zu initialisieren"

#: providers/gda-interbase-server/gda-interbase-connection.c:252
#: providers/gda-ldap-server/gda-ldap-connection.c:263
#: providers/gda-mdb-server/gda-mdb-connection.c:351
#: providers/gda-mysql-server/gda-mysql-connection.c:378
#: providers/gda-odbc-server/gda-odbc-connection.c:600
#: providers/gda-odbc-server/gda-odbc-connection.c:633
#: providers/gda-oracle-server/gda-oracle-connection.c:417
#: providers/gda-postgres-server/gda-postgres-connection.c:771
#: providers/gda-sybase-server/gda-sybase-error.c:106
#, c-format
msgid "error '%s' at %s"
msgstr "Fehler '%s' bei %s"

#: providers/gda-interbase-server/gda-interbase-connection.c:257
#: providers/gda-interbase-server/gda-interbase-connection.c:258
#: providers/gda-ldap-server/gda-ldap-connection.c:268
#: providers/gda-ldap-server/gda-ldap-connection.c:269
#: providers/gda-mdb-server/gda-mdb-connection.c:355
#: providers/gda-mdb-server/gda-mdb-connection.c:356
#: providers/gda-mysql-server/gda-mysql-connection.c:382
#: providers/gda-mysql-server/gda-mysql-connection.c:383
#: providers/gda-odbc-server/gda-odbc-connection.c:602
#: providers/gda-odbc-server/gda-odbc-connection.c:603
#: providers/gda-odbc-server/gda-odbc-connection.c:635
#: providers/gda-odbc-server/gda-odbc-connection.c:636
#: providers/gda-oracle-server/gda-oracle-connection.c:422
#: providers/gda-oracle-server/gda-oracle-connection.c:423
#: providers/gda-postgres-server/gda-postgres-connection.c:778
#: providers/gda-postgres-server/gda-postgres-connection.c:779
#: providers/gda-sybase-server/gda-sybase-error.c:115
#: providers/gda-sybase-server/gda-sybase-error.c:116
msgid "Not available"
msgstr "Nicht verfügbar"

#: providers/gda-interbase-server/gda-interbase-connection.c:259
#: providers/gda-ldap-server/gda-ldap-connection.c:270
#: providers/gda-mdb-server/gda-mdb-connection.c:357
#: providers/gda-mysql-server/gda-mysql-connection.c:384
#: providers/gda-oracle-server/gda-oracle-connection.c:424
#: providers/gda-postgres-server/gda-postgres-connection.c:780
#: providers/gda-sybase-server/gda-sybase-error.c:117
msgid "error"
msgstr "Fehler"

#: providers/gda-ldap-server/main-ldap.c:75
msgid "Could not register LDAP provider implementation"
msgstr "LDAP-Anbieter Implementierung konnte nicht registriert werden"

#: providers/gda-interbase-server/gda-interbase-connection.c:251
#, c-format
msgid "Error code %ld"
msgstr "Fehlercode %ld"

#: providers/gda-interbase-server/main-interbase.c:75
msgid "Could not register INTERBASE provider implementation"
msgstr "INTERBASE-Anbieter Implementierung konnte nicht registriert werden"

#: providers/gda-mail-server/gda-mail-connection.c:108
#: providers/gda-mdb-server/gda-mdb-connection.c:224
#: providers/gda-mysql-server/gda-mysql-connection.c:251
#: providers/gda-oracle-server/gda-oracle-connection.c:295
#: providers/gda-postgres-server/gda-postgres-connection.c:511
#: providers/gda-sybase-server/gda-sybase-connection.c:243
#, c-format
msgid "Unhandled SCHEMA_QTYPE %d"
msgstr "Unbehandelter SCHEMA_QTYPE %d"

#: providers/gda-mail-server/main-mail.c:76
msgid "Could not register Mail provider implementation"
msgstr "Die Mail-Anbieter-Implementierung konnte nicht registriert werden"

#. execute command
#: providers/gda-mdb-server/gda-mdb-command.c:86
#: providers/gda-mysql-server/gda-mysql-command.c:87
#: providers/gda-sybase-server/gda-sybase-command.c:112
#, c-format
msgid "executing command '%s'"
msgstr "Befehl '%s' ansgeführt"

#: providers/gda-mdb-server/gda-mdb-command.c:88
#: providers/gda-mysql-server/gda-mysql-command.c:89
#, c-format
msgid "info '%s'"
msgstr "Info '%s'"

#: providers/gda-mdb-server/gda-mdb-connection.c:137
#: providers/gda-mysql-server/gda-mysql-connection.c:148
#, c-format
msgid ""
"%s:%d: You cannot provide a UNIX_SOCKET if you also provide either a HOST or "
"a PORT. Please remove the UNIX_SOCKET, or remove both the HOST and PORT "
"options"
msgstr ""
"%s:%d: Sie konnen keinen UNIX_SOCKET anbieten, wenn Sie auch einen PORT und "
"einen HOST angeben. Bitten entfernen Sie entweder die PORT- und die "
"HOST-Optionen, oder entfernen Sie die UNIX_SOCKET-Optionen"

#: providers/gda-mdb-server/gda-mdb-connection.c:147
#, c-format
msgid "%s:%d: You must  provide the filename with the MS Access database"
msgstr "%s%d: Sie müssen einen Dateinamen für die MS Access Datenbank"

#: providers/gda-mdb-server/gda-mdb-connection.c:152
#, c-format
msgid "Opening connection with filename=%s, user=%s, password=%s"
msgstr "Öffne Verbindung mit Dateiname=%s, Benutzer=%s, Passwort=%s"

#: providers/gda-mdb-server/gda-mdb-connection.c:406
#: providers/gda-mysql-server/gda-mysql-connection.c:433
msgid "mysql_recset is NULL"
msgstr "mysql_recset ist NULL"

#: providers/gda-mdb-server/gda-mdb-connection.c:448
#: providers/gda-mysql-server/gda-mysql-connection.c:475
#: providers/gda-sybase-server/gda-sybase-connection.c:397
msgid "No table name given for SCHEMA_COLS"
msgstr "Kein Tabellenname für SCHEMA_COLS angegeben"

#: providers/gda-mdb-server/main-mdb.c:76
#: providers/gda-mysql-server/main-mysql.c:76
msgid "Could not register MySQL provider implementation"
msgstr "Die MySQL-Anbieter-Implementierung kann nicht registriert werden"

#: providers/gda-mysql-server/gda-mysql-connection.c:162
#, c-format
msgid ""
"Opening connection with user=%s, password=%s, host=%shost=%s, port=%s, "
"unix_socket=%s"
msgstr ""
"Öffne Verbindung mit Benutzer=%s, Passwort=%s, Rechner=%shost%s, Port=%s, "
"Unix_Socket=%s"

#: providers/gda-odbc-server/gda-odbc-connection.c:280
#, c-format
msgid "gda_odbc_connection_open_schema: Unhandled SCHEMA_QTYPE %d\n"
msgstr "gda_odbc_connection_open_schema: Unhebandelter SCHEMA_QTYPE %d\n"

#: providers/gda-odbc-server/gda-odbc-connection.c:617
#: providers/gda-odbc-server/gda-odbc-connection.c:618
#: providers/gda-odbc-server/gda-odbc-connection.c:619
#: providers/gda-odbc-server/gda-odbc-connection.c:650
#: providers/gda-odbc-server/gda-odbc-connection.c:651
#: providers/gda-odbc-server/gda-odbc-connection.c:652
msgid "no text"
msgstr "Kein Text"

#: providers/gda-odbc-server/main-odbc.c:75
msgid "Could not register ODBC provider implementation"
msgstr "Die ODBC-Anbieter-Implementierung konnte nicht registriert werden"

#: providers/gda-oracle-server/gda-oracle-command.c:195
msgid "Could not allocate statement handle"
msgstr ""

#: providers/gda-oracle-server/gda-oracle-command.c:199
msgid "command object being created without associated connection"
msgstr "Befehls-Objekt erstellt, ohne eine dazugehörige Verbindung zu haben"

#: providers/gda-oracle-server/gda-oracle-command.c:226
#, c-format
msgid "Executing command '%s'"
msgstr "Führt Befehl '%s' aus"

#: providers/gda-oracle-server/gda-oracle-connection.c:113
msgid "error in OCIHandleAlloc() HTYPE_SVCCTX"
msgstr "Fehler in OCIHandleAlloc() HTYPE_SVCCTX"

#: providers/gda-oracle-server/gda-oracle-connection.c:116
msgid "error in OCIHandleAlloc() HTYPE_SERVER"
msgstr "Fehler in OCIHandleAlloc() HTYPE_SERVER"

#: providers/gda-oracle-server/gda-oracle-connection.c:119
msgid "error in OCIHandleAlloc() HTYPE_ERROR"
msgstr "Fehler in OCIHandleAlloc() HTYPE_ERROR"

#: providers/gda-oracle-server/gda-oracle-connection.c:370
#, c-format
msgid "Unknown ORACLE type %ld"
msgstr "Unbekannter ORACLE Typ %ld"

#: providers/gda-oracle-server/gda-oracle-recordset.c:115
msgid "Unknown ORACLE data type in fill_field_values"
msgstr "Unbekannter ORACLE Datentyp in fill_field_values"

#: providers/gda-oracle-server/gda-oracle-recordset.c:150
msgid "recordset object being created without associated connection"
msgstr "Recordset-Objekt erstellt, ohne eine dazugehörige Verbindung zu haben"

#: providers/gda-oracle-server/main-oracle.c:75
msgid "Could not register ORACLE provider implementation"
msgstr "Die ORACLE-Anbieter-Implementiertung konnte nicht registriert werden"

#: providers/gda-postgres-server/gda-postgres-connection.c:880
#: providers/gda-postgres-server/gda-postgres-connection.c:992
msgid "postgres_recset is NULL"
msgstr "postgres_recset ist NULL"

#: providers/gda-postgres-server/main-postgres.c:75
msgid "Could not register PostgreSQL provider implementation"
msgstr ""
"Die PostgreSQL-Anbieter-Implementiertung konnte nicht registriert werden"

#: providers/gda-sybase-server/gda-sybase-command.c:107
msgid "Connection seems to be dead"
msgstr "Die Verbindung scheint tod zu sein"

#: providers/gda-sybase-server/gda-sybase-command.c:127
msgid "Filling command structure failed\n"
msgstr "Füllen der Befehlsstruktur gescheitert\n"

#: providers/gda-sybase-server/gda-sybase-command.c:139
msgid "sending cmd failed"
msgstr "Senden von cmd gescheitert"

#: providers/gda-sybase-server/gda-sybase-command.c:146
msgid "Allocating recordset failed"
msgstr "Erreichen des Recordset gescheitert"

#: providers/gda-sybase-server/gda-sybase-command.c:156
msgid "Getting recordset userdata failed"
msgstr "Erreichen der Recordset Benutzerdaten gescheitert"

#: providers/gda-sybase-server/gda-sybase-command.c:212
msgid "Command execution finished, all results processed"
msgstr "Befehlsausführung beendet, alle Ergebnisse weitergeleitet"

#: providers/gda-sybase-server/gda-sybase-command.c:216
msgid "Command failed"
msgstr "Befehl gescheitert"

#: providers/gda-sybase-server/gda-sybase-command.c:220
msgid "Execution of resultless command succeeded"
msgstr "Ausführen des Ergebnisslosen Befehls gescheitert"

#: providers/gda-sybase-server/gda-sybase-command.c:236
#, c-format
msgid "ct_results() returned %x"
msgstr "ct_results() gab %x zurück"

#: providers/gda-sybase-server/gda-sybase-command.c:246
msgid "No further results, command completly succeeded."
msgstr "Keine weiteren Ergebnisse, Befehl erfolgreich beendet."

#: providers/gda-sybase-server/gda-sybase-command.c:250
msgid "An error occured during command execution."
msgstr "Ein Fehler trat beim Ausführen des Befehls auf."

#: providers/gda-sybase-server/gda-sybase-command.c:257
msgid "Cmd execution returned unknown result, may be an error"
msgstr ""
"Ausführen des Befehls hat ein unbekanntes Ergebnis zurückgegeben, "
"möglicherweise ein Fehler"

#: providers/gda-sybase-server/gda-sybase-command.c:270
msgid "No data returned, freeing recordset structure"
msgstr "Kine Daten zurückgegeben, lösche die Struktur"

#: providers/gda-sybase-server/gda-sybase-command.c:301
#: providers/gda-sybase-server/gda-sybase-command.c:313
msgid "Freeing command structure failed"
msgstr "Löschen der Befehls-Struktur gescheitert"

#: providers/gda-sybase-server/gda-sybase-connection.c:123
msgid "Error allocating error structure"
msgstr "Fehler beim Erreichen der Fehlerstruktur"

#: providers/gda-sybase-server/gda-sybase-connection.c:131
msgid "Not enough memory for sybase_Connection"
msgstr "Nicht genug Speicher für sybase_Connection"

#: providers/gda-sybase-server/gda-sybase-connection.c:387
#, c-format
msgid "schema_cols: table_name '%s'"
msgstr "schema_cols: table_name '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:403
msgid "Could not allocate command structure"
msgstr "Konnte die Befehlsstruktur nicht erreichen"

#: providers/gda-sybase-server/gda-sybase-connection.c:425
msgid "Could not allocate enough memory for query"
msgstr "Es konnte nicht genug Speicher für die Anfrage reserviert werden"

#: providers/gda-sybase-server/gda-sybase-connection.c:439
msgid "Could not get recordset in schema_cols"
msgstr "Konnte den Recordset in schema_cols nicht erreichen"

#: providers/gda-sybase-server/gda-sybase-connection.c:471
#: providers/gda-sybase-server/gda-sybase-connection.c:548
#, c-format
msgid "schema_tables: table name = '%s'"
msgstr "schema_tables: Tabellenname = '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:479
#: providers/gda-sybase-server/gda-sybase-connection.c:556
#, c-format
msgid "schema_tables: table_schema = '%s'"
msgstr "schema_tables: table_schema = '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:484
#, c-format
msgid "schema_procedures: proc_catalog = '%s' UNUSED!\n"
msgstr "schema_procedures: proc_catalog = '%s' UNGENUTZT!\n"

#: providers/gda-sybase-server/gda-sybase-connection.c:488
#, c-format
msgid "schema_procs: invalid constraint type %d"
msgstr "schema_procs: Ungültiger Konstantentyp %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:498
msgid "Could not allocate cmd structure"
msgstr "Die Befehls-Struktur konnte nicht erreicht werden"

#: providers/gda-sybase-server/gda-sybase-connection.c:561
#, c-format
msgid "schema_procedures: proc_catalog = '%s' UNUSED!"
msgstr "schema_procedures: proc_catalog = '%s' UNGENUTZT!"

#: providers/gda-sybase-server/gda-sybase-connection.c:565
#, c-format
msgid "schema_tables: invalid constraint type %d"
msgstr "schema_tables: Ungültiger Konstantentyp %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:626
#, c-format
msgid "Invalid or unsupported constraint type: %d"
msgstr "Ungültiger oder nicht unterstützter Konstantentyp: %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:667
msgid "Could not allocate Gda_ServerCommand"
msgstr "Konnte Gda_ServerCommand nicht erreichen"

#: providers/gda-sybase-server/gda-sybase-connection.c:697
#: providers/gda-sybase-server/gda-sybase-connection.c:702
#, c-format
msgid "connection status: %x"
msgstr "Verbindungsstatus: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:698
#, c-format
msgid "return code: %x"
msgstr "Rückgabewert: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:735
msgid "attempting reconnection"
msgstr "Versuche Wiederverbindung"

#: providers/gda-sybase-server/gda-sybase-connection.c:872
#, c-format
msgid "%sonnected to '%s'"
msgstr "%sVerbunden mit '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:882
#, c-format
msgid "Selected database '%s'"
msgstr "Wählt Datenbank '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:911
#, c-format
msgid "%s: Could not allocate cmd structure"
msgstr "%s: Di Befehls-Struktur konnte nicht erreicht werden"

#: providers/gda-sybase-server/gda-sybase-connection.c:920
#, c-format
msgid "%s: Could not allocate memory for cmd string"
msgstr "%s: Konnte keinen Speicher für den Befehlsstring erreichen"

#: providers/gda-sybase-server/gda-sybase-connection.c:933
#, c-format
msgid "%s: ct_command failed"
msgstr "%s: ct_command gescheitert"

#: providers/gda-sybase-server/gda-sybase-connection.c:942
#, c-format
msgid "%s: sending command failed"
msgstr "%s: Senden des Befehls gescheitert"

#: providers/gda-sybase-server/gda-sybase-connection.c:953
#, c-format
msgid "%s: selecting database failed"
msgstr "%s: Auswählen der Datenbank gescheitert"

#: providers/gda-sybase-server/gda-sybase-connection.c:971
#, c-format
msgid "%s: use command failed"
msgstr "%s: use-Befehl gescheitert"

#: providers/gda-sybase-server/gda-sybase-connection.c:998
msgid "Could not allocate locale structure"
msgstr "Die lokale Struktur kann nicht erreicht werden"

#: providers/gda-sybase-server/gda-sybase-connection.c:1004
#, c-format
msgid "Could not set locale to '%s'"
msgstr "lokale Struktur konnte nicht auf '%s' gesetzt werden"

#: providers/gda-sybase-server/gda-sybase-connection.c:1010
#, c-format
msgid "Could not configure locale to be '%s'"
msgstr ""
"Die lokale Struktur konnte nicht so eingestellt werden, dass sie '%s' ist"

#: providers/gda-sybase-server/gda-sybase-connection.c:1015
msgid "Could not drop locale structure"
msgstr "Konnte die lokale Struktur nicht fallenlassen"

#: providers/gda-sybase-server/gda-sybase-connection.c:1028
#, c-format
msgid "Fatal error: %s\n"
msgstr "Schwerer Fehler: %s\n"

#: providers/gda-sybase-server/gda-sybase-error.c:62
#, c-format
msgid "gda_sybase_error_make(%p,%p,%p,%p)"
msgstr "gda_sybase_error_make(%p,%p,%p,%p)"

#: providers/gda-sybase-server/gda-sybase-error.c:71
#, c-format
msgid "No error flag ist set, though %s is called. This may be a bug."
msgstr ""
"Kein Fehlercode angegeben, obwohl %s aufgerufen wurde. Die könnte ein Bug "
"sein."

#: providers/gda-sybase-server/gda-sybase-error.c:135
msgid "cs_diag() initialized"
msgstr "cs_diag() initialisiert"

#: providers/gda-sybase-server/gda-sybase-error.c:138
msgid "Could not initialize cs_diag()"
msgstr "cs_diag() konnte nicht initialisiert werden"

#: providers/gda-sybase-server/gda-sybase-error.c:145
msgid "ct_diag() initialized"
msgstr "ct_diag() initialisiert"

#: providers/gda-sybase-server/gda-sybase-error.c:148
msgid "Could not initialize ct_diag()"
msgstr " ct_diag()konnte nicht initialisiert werden"

#: providers/gda-sybase-server/gda-sybase-error.c:182
#, c-format
msgid "%s could not find server implementation"
msgstr "%s konnte die Server-Implementierung nicht finden"

#: providers/gda-sybase-server/gda-sybase-error.c:188
#, c-format
msgid "%s could not get cnc userdata"
msgstr "%s konnte die cnc-Benutzerdaten nicht erhalten"

#: providers/gda-sybase-server/gda-sybase-error.c:200
msgid "Could not get # of messages"
msgstr "Die Zahl der Meldungen konnte nicht erhalten werden"

#: providers/gda-sybase-server/gda-sybase-error.c:205
#, c-format
msgid "%d ctlib messages to fetch"
msgstr "%d ctlib-Meldungen zu Empfangen"

#: providers/gda-sybase-server/gda-sybase-error.c:207
#, c-format
msgid "%d ctlib message to fetch"
msgstr "ctlib-Meldung %d zu Empfangen"

#: providers/gda-sybase-server/gda-sybase-error.c:243
msgid "error is nullpointer in gda_sybase_messages_handle"
msgstr "Fehler ist nullpointer in gda_sybase_messages_handle"

#: providers/gda-sybase-server/gda-sybase-error.c:268
#: providers/gda-sybase-server/gda-sybase-error.c:335
#: providers/gda-sybase-server/gda-sybase-error.c:378
msgid "Could not get # of server messages"
msgstr "Die Zahl der Server-Meldungen konnte nicht erhalten werden"

#: providers/gda-sybase-server/gda-sybase-error.c:276
#, c-format
msgid "Fetching %d client messages"
msgstr "Empfange %d Client-Meldungen"

#: providers/gda-sybase-server/gda-sybase-error.c:278
#, c-format
msgid "Fetching %d client message"
msgstr "Empfange Client-Meldung %d"

#: providers/gda-sybase-server/gda-sybase-error.c:287
#, c-format
msgid "Could not get client msg # %d"
msgstr "Die Client-Meldungen %d konnte nicht empfangen werden"

#: providers/gda-sybase-server/gda-sybase-error.c:303
msgid "Could not get # of client messages"
msgstr "Die Zahl der Client-Meldungen konnte nicht ermittelt werden"

#: providers/gda-sybase-server/gda-sybase-error.c:311
msgid "Could not clear client messages"
msgstr "Konnte die Client-Meldungen nicht löschen"

#: providers/gda-sybase-server/gda-sybase-error.c:343
#, c-format
msgid "Fetching %d server messages"
msgstr "Empfange %d Server-Meldungen"

#: providers/gda-sybase-server/gda-sybase-error.c:345
#, c-format
msgid "Fetching %d server message"
msgstr "Empfange Server-Mendung %d"

#: providers/gda-sybase-server/gda-sybase-error.c:354
#, c-format
msgid "Could not get server msg # %d"
msgstr "Die Server-Meldung %d konnte nicht empfangen werden"

#: providers/gda-sybase-server/gda-sybase-error.c:386
msgid "Could not clear server messages"
msgstr "Die Server-Meldungen konnte nicht gelöscht werden"

#: providers/gda-sybase-server/gda-sybase-error.c:408
#: providers/gda-sybase-server/gda-sybase-error.c:444
msgid "Could not get # of cslib messages"
msgstr "Die Zahl der cslib Meldungen konnte nicht empfangen werden"

#: providers/gda-sybase-server/gda-sybase-error.c:416
#, c-format
msgid "Fetching %d cslib messages"
msgstr "Empfage %d cslib-Meldungen"

#: providers/gda-sybase-server/gda-sybase-error.c:418
#, c-format
msgid "Fetchong %d cslib message"
msgstr "Empfagne cslib-Meldung %d"

#: providers/gda-sybase-server/gda-sybase-error.c:428
#, c-format
msgid "Could not get cslib msg # %d"
msgstr "Die cslib-Meldung %d konnte nicht empfagen werden"

#: providers/gda-sybase-server/gda-sybase-error.c:452
msgid "Could not clear cslib messages"
msgstr "Die cslib-Meldungen konnte nicht gelöscht werden"

#: providers/gda-sybase-server/gda-sybase-error.c:584
msgid "Server message:"
msgstr "Server-Nachricht:"

#: providers/gda-sybase-server/gda-sybase-error.c:593
msgid "Client message:"
msgstr "Client-Nachricht:"

#: providers/gda-sybase-server/gda-sybase-error.c:600
msgid "CS-Library error:"
msgstr "CS-Bibliotheks-Fehler:"

#: providers/gda-sybase-server/gda-sybase-recordset.c:148
#, c-format
msgid "fetch error at row %d"
msgstr "Empfing Fehler in Zeile %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:155
#, c-format
msgid "CS_END_DATA on row %d; canceling"
msgstr "CS_END_DATA in Zeile %d; breche ab"

#: providers/gda-sybase-server/gda-sybase-recordset.c:164
#, c-format
msgid "%s: Error (%d)"
msgstr "%s: Fehler (%d)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:339
msgid "Failed fetching # of columns"
msgstr "Die Zahl der Spalten konnte nicht empfagen werden"

#: providers/gda-sybase-server/gda-sybase-recordset.c:345
#, c-format
msgid "Counted %d columns"
msgstr "%d Spalten gezählt"

#: providers/gda-sybase-server/gda-sybase-recordset.c:352
#, c-format
msgid "%s could not allocate datafmt info"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-recordset.c:362
#, c-format
msgid "%s could not allocate data fields"
msgstr "%s konnte die Datenfelder nicht erreichen"

#: providers/gda-sybase-server/gda-sybase-recordset.c:380
#, c-format
msgid "ct_describe on col # %d"
msgstr "ct_describe in Zeile %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:387
#, c-format
msgid "ct_describe on col # %d failed"
msgstr "ct_describe in Zeile %d gescheitert"

#: providers/gda-sybase-server/gda-sybase-recordset.c:393
#, c-format
msgid "ct_describe(%d): '%s', %d (%s)"
msgstr "ct_describe(%d): '%s', %d (%s)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:402
msgid "could not allocate data holder"
msgstr "Der Datenhalter konnte nicht erreicht werden"

#: providers/gda-sybase-server/gda-sybase-recordset.c:412
msgid "could not ct_bind data holder to column"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-recordset.c:419
msgid "illegal type detected, aborting"
msgstr "Illegalen Typ entdeckt, breche ab"

#: providers/gda-sybase-server/gda-sybase-recordset.c:425
msgid "could not allocate field"
msgstr "Das Feld konnte nicht erreicht werden"

#: providers/gda-sybase-server/main-sybase.c:80
msgid "Could not register gda-sybase provider implementation"
msgstr ""
"Die gda-sybase-Anbieter-Implementiertung konnte nicht registriert werden"

#: tools/gda-run.c:29
msgid "database name"
msgstr "Datenbankname"

#: tools/gda-run.c:29
msgid "DATABASE"
msgstr "DATENBANK"

#: tools/gda-run.c:30
msgid "user name"
msgstr "Benutzername"

#: tools/gda-run.c:30
msgid "USER-NAME"
msgstr "BENUTZERNAME"

#: tools/gda-run.c:31
msgid "password"
msgstr "Passwort"

#: tools/gda-run.c:31
msgid "PASSWORD"
msgstr "PASSWORT"

#: tools/gda-run.c:32
msgid "file name"
msgstr "Dateiname"

#: tools/gda-run.c:32
msgid "FILE-NAME"
msgstr "DATEINAME"

#: tools/gda-run.c:104
#, c-format
msgid "%s: misconfigured DSN entry"
msgstr "%s: Fehlkonfigurierter DSN Eintrag"

#: tools/gda-run.c:119
#, c-format
msgid "%s: data source '%s' not found\n"
msgstr "%s: Datenquelle '%s' nicht gefunden\n"

